design/numeral/styles.css

```diff
@@ -0,0 +1,51 @@
+:root {
+  --numeral-value-sm-size: 2em;
+  --numeral-value-md-size: 3em;
+  --numeral-value-lg-size: 4em;
+  --numeral-value-size: var(--numeral-value-md-size);
+}
+
+.container {
+  /* For the future */
+}
+
+.caption {
+  display: block;
+}
+
+.appendix {
+  font-size: calc(0.75 * var(--numeral-value-size, 2em));
+  font-weight: var(--numeral-value-weight, 300);
+  line-height: 1;
+  vertical-align: baseline;
+}
+
+.value {
+  font-size: var(--numeral-value-size, 2em);
+  font-weight: var(--numeral-value-weight, 300);
+  line-height: 1;
+}
+
+.diff {
+  vertical-align: top;
+}
+
+.diff-none {
+  display: none;
+}
+
+.diff-positive::before {
```

Comment:
these classes are not used

Reply:
Yet, we do not need it for this moment because it's overkill for this task.

Reply:
Can you remove them then?